docs/insights-archive-sample/customresources/core.openstack.org/openstackcontrolplanes/openstack/openstack-galera-network-isolation.json

```diff
@@ -0,0 +1,1597 @@
+{
+  "apiVersion": "core.openstack.org/v1beta1",
+  "kind": "OpenStackControlPlane",
+  "metadata": {
+    "annotations": {},
+    "creationTimestamp": "2024-06-10T14:09:17Z",
+    "finalizers": [
+      "OpenStackControlPlane"
```

Comment:
when you get chance can update with current latest as it's outdated after recent finalizer changes.

Reply:
what do you mean exactly? I see that finalizers list is like that in my dev env

Reply:
you likely not have pulled latest operator changes that's why seeing old content

Reply:
the format we were using to generate the filnaliser list was incorrect in lib common but it was not being validated by k8s that i belive has been fix in a new version so we fixed lib-common to do this properly 

https://github.com/openstack-k8s-operators/lib-common/pull/519
https://github.com/openstack-k8s-operators/neutron-operator/pull/366

so it changes form "OpenStackControlPlane" the kind to fqcn

openstack.org/openstackcontrolplane

Reply:
Done